build.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+
+set -e
+
+repo=${repo:-t184256/nix-on-droid-bootstrap}  # set this to your fork!
+branch=${branch:-$(git rev-parse --abbrev-ref HEAD)}
+
+arches='aarch64 i686'
+nixOnDroidChannelURL=https://github.com/$repo/archive/$branch.tar.gz
+
+mkdir -p out
+rm -f out/*
+for arch in $arches; do
+	echo $arch: building talloc...
+	nix build --show-trace -f pkgs --argstr arch $arch tallocStatic -o out/talloc-$arch
+	talloc=$(realpath out/talloc-$arch)
+
+	echo $arch: building proot...
+	nix build --show-trace -f pkgs --argstr arch $arch prootTermux -o out/proot-$arch
+	proot=$(realpath out/proot-$arch)
+
+	echo $arch: patching proot path in modules/environment/login/default.nix...
+	grep "$arch = \"/nix/store/" modules/environment/login/default.nix
+	sed -i "s|$arch = \"/nix/store/.*\";|$arch = \"$proot\";|" modules/environment/login/default.nix
+	grep "$arch = \"/nix/store/" modules/environment/login/default.nix
+
+	echo $arch: building nix-on-droid...
+	nix build --show-trace -f pkgs --argstr arch $arch --argstr nixOnDroidChannelURL $nixOnDroidChannelURL bootstrapZip -o out/nix-on-droid-$arch
+
+	echo $arch: injecting talloc/proot for initial bootstrap...
+
+	cat out/nix-on-droid-$arch/bootstrap-$arch.zip > out/bootstrap-$arch.zip
+	nix-store --export --readonly-mode $talloc > out/talloc-$arch.closure
+	nix-store --export --readonly-mode $proot > out/proot-$arch.closure
```

Comment:
Why not getting the complete closure? This way you do not need to export talloc manually
```suggestion
	nix-store --export --readonly-mode $(nix-store -qR $proot) > out/proot-$arch.closure
```

Reply:
Because I didn't know that trick!